debian/rules

```diff
@@ -23,6 +24,21 @@ else
 endif
 endif
 
+override_dh_gencontrol:
+ifeq ($(DIST_RELEASE),trusty)
+	echo "extra:Depends=libapt-pkg4.12 (>= 1.0.1ubuntu2.23)" >> debian/ubuntu-advantage-tools.substvars
+else ($(DIST_RELEASE),xenial))
+	echo "extra:Depends=libapt-pkg5.0 (>= 1.2.30)" >> debian/ubuntu-advantage-tools.substvars
+else ($(DIST_RELEASE),cosmic))
+	echo "extra:Depends=libapt-pkg5.0 (>= 1.7.3)" >> debian/ubuntu-advantage-tools.substvars
+else ($(DIST_RELEASE),disco))
+	echo "extra:Depends=libapt-pkg5.0 (>= 1.8.0~rc1)" >> debian/ubuntu-advantage-tools.substvars
```

Comment:
Just 1.8.0 perhaps, instead of ~rc1? https://launchpad.net/ubuntu/+source/apt/1.8.0 I don't see rc1 published anywhere.

Reply:
moving disco to 1.8.1 since that's in disco-updates  (and dropping the ifeq disco because eoan is on the same minimim version so we don't need to break out disco as a separate conditional from >= eoan.